tests/infrastructure/prometheus.go

```diff
@@ -440,9 +441,11 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", Serial, decorators.SigCo
 					req, _ := http.NewRequest("GET", metricsURL, nil)
 					resp, err := client.Do(req)
 					if err != nil {
-						fmt.Fprintf(GinkgoWriter, "client: request: %v #%d: %v\n", req, ix, err) // troubleshooting helper
+						_, err := fmt.Fprintf(GinkgoWriter, "client: request: %v #%d: %v\n", req, ix, err) // troubleshooting helper
```

Comment:
Please attempt to avoid shadowing the err, it is confusing.

Reply:
Oh, nice catch!